db/migrate/20170616053125_add_notification_settings_for_users.rb

```diff
@@ -0,0 +1,18 @@
+class AddNotificationSettingsForUsers < ActiveRecord::Migration
+  def change
+    all_users = User.all
+    all_users_notifications = []
+    all_users.each do |user|
+      NotificationSetting::NOTIFICATION_TYPES.each do |st|
+        all_users_notifications << {
+          setting_type: st,
+          user: user,
+          fb_messenger_enabled: false,
+          mobile_enabled: false,
+          email_enabled: false
+        }
+      end
+    end
+    NotificationSetting.create(all_users_notifications)
```

Comment:
This is probably a huge RAM hog, since it's gonna create 800,000-1,000,000 hashes in RAM.  Probably better to do smaller batches, maybe move this into the outer loop so it flushes after each user